wait/host.go

```diff
@@ -587,24 +587,32 @@ func (a *HostAwaitility) WaitForTemplateUpdateRequests(namespace string, count i
 type NotificationWaitCriterion func(a *HostAwaitility, mur *toolchainv1alpha1.Notification) bool
 
 // WaitForNotification waits until there is a Notification available with the given name and the optional conditions
-func (a *HostAwaitility) WaitForNotification(name string, criteria ...NotificationWaitCriterion) (*toolchainv1alpha1.Notification, error) {
-	var notification *toolchainv1alpha1.Notification
+func (a *HostAwaitility) WaitForNotification(name string, criteria ...NotificationWaitCriterion) ([]*toolchainv1alpha1.Notification, error) {
```

Comment:
since this func will return a `[]*toolchainv1alpha1.Notification`, shouldn't we rename it to `WaitForNotifications` (plural)?

Reply:
Good point! Updated it as part of this commit: fd66ead0da803a9206a4e0326c56ae73dec1d2a5